pages/survey.jsx

```diff
@@ -23,3 +24,5 @@ module.exports = React.createClass({
     );
   }
 });
+
+module.exports = Survey;
```

Comment:
As I state in mofo eng call that doing it this new way is better
On Mar 15, 2016 6:02 PM, "ScottDowne" <notifications@github.com> wrote:

> In pages/survey.jsx
> <https://github.com/mozilla/donate.mozilla.org/pull/1397#discussion_r56248972>
> :
>
> > @@ -23,3 +24,5 @@ module.exports = React.createClass({
> >      );
> >    }
> >  });
> > +
> > +module.exports = Survey;
>
> module.exports = React.createClass is no longer supported?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/donate.mozilla.org/pull/1397/files/32b2f5f363b33089041b9468eb5fdfd9bb7080a1#r56248972>
>